src/app.py

```diff
@@ -2869,7 +2869,7 @@ def _get_conflicting_apps(domain, path, ignore_app=None):
         for p, a in apps_map[domain].items():
             if a["id"] == ignore_app:
                 continue
-            if path == p or path == "/" or p == "/":
+            if path == p or ( not path.startswith("/.well-known/") and path == "/" ) or ( not path.startswith("/.well-known/") and p == "/" ):
```

Comment:
```suggestion
            if path == p or ( not path.startswith("/.well-known/") and ( path == "/" or p == "/" ) ):
```

A bit of factorization